Classes/Providers/OTT/Model/OTTAsset.swift

```diff
@@ -11,29 +11,28 @@ import SwiftyJSON
 
 internal class OTTAsset: OTTBaseObject {
 
-    internal var id: String 
+    internal var id: String
```

Comment:
internal access level is by default and doesn't need to be add, please remove all internal modifiers.